modules/afsocket/transport-unix-socket.c

```diff
@@ -34,6 +34,8 @@
 #include <errno.h>
 #include <unistd.h>
 
+#pragma GCC diagnostic ignored "-Wunused-function"
```

Comment:
I would really not like to follow the path of pragmas.

Reply:
I could resolve this with adding G_GNUC_UNUSED to the conflicting function definitions.